.github/workflows/lint-and-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: lint-and-build
+on: push
+env:
+  DOCKER_REGISTRY: docker.io
+  DOCKER_REPOSITORY: gcloud-kubectl-helm
+  # set this in Github repository -> setting -> Secrets and Variables
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  # should be set automatic by Github actions
+  #GITHUB_REPOSITORY_OWNER:
+jobs:
+  lint-scripts:
+    runs-on: ubuntu-latest
+    container:
+      image: koalaman/shellcheck-alpine
+    steps:
+    - name: Dump GitHub context
+      id: github_context_step
+      run: echo '${{ toJSON(github) }}'
```

Comment:
doesn't this leak confidential data in the logs?

Reply:
As far as I could see - not :-)